ci/Dockerfile

```diff
@@ -1,62 +1,140 @@
-FROM a12e/docker-qt:5.14-gcc_64
+ARG QT_VERSION=5.15.2
+
+# QT Installation Image --------------------------------------------------------
+FROM ubuntu:18.04 AS qt-install
+
+ARG QT_VERSION
+
+RUN apt update && apt full-upgrade -y && apt install -y --no-install-recommends \
+    sudo python3 python3-pip \
+    && apt-get -qq clean
+
+RUN python3 -m pip install setuptools
+
+RUN chmod -R 777 /opt
+
+RUN groupadd -r user && useradd --create-home --gid user user && echo 'user ALL=NOPASSWD: ALL' > /etc/sudoers.d/user
+
+
+COPY scripts/install-qt.sh /home/user/
+RUN chown user:user /home/user/install-qt.sh
+RUN chmod a+x /home/user/install-qt.sh
+
+USER user
+WORKDIR /home/user
+
+ENV HOME /home/user
+ENV QT_VERSION=$QT_VERSION
+
+# Install QT
+RUN /home/user/install-qt.sh
+
+
+
+# Build Image ------------------------------------------------------------------
+FROM ubuntu:18.04
+
+ARG LINUXDEPLOYQT_VERSION=continuous
+ARG QT_VERSION
+
+
+# Adapted from a12e/docker-qt by Aurélien Brooke
+
+ENV DEBIAN_FRONTEND=noninteractive \
+    QMAKESPEC=linux-g++ \
+    QT_PATH=/opt/qt \
+    QT_PLATFORM=gcc_64 \
+    QT_VERSION=$QT_VERSION
+
+ENV \
+    PATH=${QT_PATH}/${QT_VERSION}/${QT_PLATFORM}/bin:$PATH
+
+
+RUN apt update -yq && apt install -yq software-properties-common \
+&& add-apt-repository -y ppa:git-core/ppa \
+&& add-apt-repository -y ppa:ubuntu-toolchain-r/test \
+&& apt update -yq && apt full-upgrade -yq && apt install -yq --no-install-recommends --fix-missing  \
+    git \
+    openssh-client \
+    ca-certificates \
+    locales \
+    sudo \
+    curl \
+    build-essential \
+    pkg-config \
+    libgl1-mesa-dev \
+    libsm6 \
+    libice6 \
+    libxext6 \
+    libxrender1 \
+    libfontconfig1 \
+    libdbus-1-3 \
+    libz-dev \
+    fuse \
+    file \
+    libxkbcommon-dev \
+    libxkbcommon-x11-dev \
+    wget \
+    make libssl-dev zlib1g-dev \
+    libbz2-dev libreadline-dev libsqlite3-dev llvm \
+    libncurses5-dev libncursesw5-dev xz-utils tk-dev \
+    libgstreamer-plugins-base1.0-0 libpcsclite-dev \
+    cmake extra-cmake-modules s3cmd libpcre3-dev libnss3 libxcomposite1 libxtst6 jq \
+    gstreamer1.0-plugins-good gstreamer1.0-plugins-bad gstreamer1.0-plugins-ugly \ 
+    gstreamer1.0-libav gstreamer1.0-tools gstreamer1.0-alsa libpulse-mainloop-glib0 \
+    gstreamer1.0-pulseaudio libxrandr2 libxcursor1 libxi6 libxcb-icccm4 libxcb-image0 \
+    libxcb-keysyms1 libxcb-render-util0 libxcb-render0 libxcb-xinerama0 \
+    gnupg2 gcc-9 g++-9 \
+&&  update-alternatives \
+    --install /usr/bin/gcc gcc /usr/bin/gcc-9 100 \
+    --slave /usr/bin/g++ g++ /usr/bin/g++-9 \
+    && apt-get -qq clean
+
+COPY scripts/* /tmp/build/scripts/
+RUN chmod a+x /tmp/build/scripts/*.sh
+
+# Download & install linuxdeployqt
+RUN /tmp/build/scripts/install-linuxdeployqt.sh
```

Comment:
Same thing here. Youre adding this script, adjusting permissions and then running it, just to do also 3 things:
https://github.com/status-im/status-desktop/blob/fb28070e340bafc5258a6228dd66b2cfe326c8b0/ci/scripts/install-linuxdeployqt.sh#L13-L15
So might as well do it here and drop the whole `ci/scripts` directory.